lib/mayaUsd/fileio/primUpdater.h

```diff
@@ -49,71 +52,64 @@ class UsdMayaPrimUpdater
         Push = 1 << 0,
```

Comment:
Do we still use Push,Pull,Clear anywhere?

Reply:
Pull and Clear are not used currently in `dev`, but are integral to #1759 for allowing the mayaUsdProxyShape to support Maya references (see `mayaReferenceUpdater.cpp`) If they're removed by this PR they'll just get added back in later.